flake.nix

```diff
@@ -208,6 +210,37 @@
         };
       };
 
+      buildPlutipRuntime = system: extraConfig:
```

Comment:
Is the idea here to eventually add Plutip as a service?

Reply:
`plutip-server`. Yes, there's even a task for that: https://github.com/Plutonomicon/cardano-transaction-lib/issues/507